.github/workflows/ci.yml

```diff
@@ -43,7 +43,8 @@ jobs:
         # also test 'Release'.
         build_testing: [ON, OFF]
         exclude:
-          # Currently our tests don't compile in Windows so skip this.
+          # Currently our tests don't compile, and nor do we have a coverage
+          # build in Windows so skip this.
```

Comment:
:grimacing: Should this _maybe_ be an issue or are we committing to not supporting Windows?

Reply:
I'm inclined to say no. I softly doubt any of Peter's future developers will be using Windows.
Could tack this on to...
* #41 

...?

Reply:
It's a completely different tool to `gcov`, though. Also, I don't know how to run it... maybe Mosè does?

Reply:
Sorry, what is "It" here?

Reply:
I guess I mean [this Microsoft.CodeCoverage.Console](https://learn.microsoft.com/en-us/visualstudio/test/microsoft-code-coverage-console-tool?view=vs-2022) tool.